tests/ident/test_engine_parser_comet_2020_01_04.py

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env python
 
-import pandas as pd
 import pytest
 from lxml import etree
```

Comment:
now that lxml isnt used anymore we should drop this in favour of the python implementation